### ISSUE TITLE
Xamarin iOS application crashes after hover mouse over System.DateTime

### ISSUE THREAD

**Issue by rolfbjarne** [dotnet/runtime#98949]
_From @nevse on Fri, 23 Feb 2024 11:08:50 GMT_

### Steps to Reproduce

1. Launch any application via vscode with debugger.
2. Set breakpoint in some code that contains System.DateTime type
3. Hover mouse on the type/variable with DateTime while you stopped at breakpoint and app will crash

### Expected Behavior
App continues execution and show variable value of DateTime type

### Actual Behavior
App crash with assertion exception

### Environment

MacOS 14, .net 8.0.102, vscode + .net meteor extension (maui extension same problem)

### Build Logs

```
2024-02-23 14:56:06.082531+0400 Maui.TestApp[23957:270582] error: * Assertion at /Users/runner/work/1/s/src/mono/mono/component/debugger-agent.c:6504, condition `res' not met
=================================================================
Native Crash Reporting
=================================================================
Got a SIGABRT while executing native code. This usually indicates
a fatal error in the mono runtime or one of the native libraries
used by your application.
=================================================================

=================================================================
Native stacktrace:
=================================================================
0x1033601a0 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_dump_native_crash_info
0x1033101c0 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_handle_native_crash
0x103632bc8 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : sigabrt_signal_handler.cold.1
0x10335fa60 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_runtime_setup_stat_profiler
0x101bc7c60 - /usr/lib/system/libsystem_platform.dylib : _sigtramp
0x1027231d0 - /usr/lib/system/libsystem_pthread.dylib : pthread_kill
0x1801605cc - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/usr/lib/system/libsystem_c.dylib : abort
0x101c31914 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmono-component-debugger.dylib : monoeg_log_set_default_handler
0x101c31658 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmono-component-debugger.dylib : monoeg_g_logv_nofree
0x101c316fc - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmono-component-debugger.dylib : monoeg_assertion_message
0x101c31740 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmono-component-debugger.dylib : mono_assertion_message_unreachable
0x101c16bf8 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmono-component-debugger.dylib : mono_do_invoke_method
0x101c13220 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmono-component-debugger.dylib : suspend_current
0x101c1575c - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmono-component-debugger.dylib : process_event
0x101c14ee4 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmono-component-debugger.dylib : mono_dbg_process_breakpoint_events
0x101c29e38 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmono-component-debugger.dylib : mono_de_process_breakpoint
0x101c20548 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmono-component-debugger.dylib : debugger_agent_breakpoint_from_context
0x1006d09e4 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/Maui.TestApp : sdb_breakpoint_trampoline
0x10336e548 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : do_debugger_tramp
0x10336c6d0 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_interp_exec_method
0x103361910 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : interp_runtime_invoke
0x10327a9b0 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_jit_runtime_invoke
0x103435a14 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_runtime_invoke_checked
0x10343eff8 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_runtime_try_invoke_byrefs
0x1033f47d0 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : ves_icall_InternalInvoke
0x1033ffec4 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : ves_icall_InternalInvoke_raw
0x103370a1c - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : do_icall
0x10336f328 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : do_icall_wrapper
0x103363d3c - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_interp_exec_method
0x103360fd8 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : interp_entry_from_trampoline
0x1006d5a18 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/Maui.TestApp : native_to_interp_trampoline
0x18016cd3c - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/usr/lib/system/libdispatch.dylib : _dispatch_client_callout
0x1801706d4 - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/usr/lib/system/libdispatch.dylib : _dispatch_continuation_pop
0x180183e6c - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/usr/lib/system/libdispatch.dylib : _dispatch_source_invoke
0x18017b998 - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/usr/lib/system/libdispatch.dylib : _dispatch_main_queue_drain
0x18017b61c - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/usr/lib/system/libdispatch.dylib : _dispatch_main_queue_callback_4CF
0x1803f1a30 - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation : __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__
0x1803ec148 - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation : __CFRunLoopRun
0x1803eb5a4 - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation : CFRunLoopRunSpecific
0x18e9fbae4 - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/System/Library/PrivateFrameworks/GraphicsServices.framework/GraphicsServices : GSEventRunModal
0x1852f02e4 - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/System/Library/PrivateFrameworks/UIKitCore.framework/UIKitCore : -[UIApplication _run]
0x1852f3f5c - /Library/Developer/CoreSimulator/Volumes/iOS_21C62/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 17.2.simruntime/Contents/Resources/RuntimeRoot/System/Library/PrivateFrameworks/UIKitCore.framework/UIKitCore : UIApplicationMain
0x1027b3f44 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libxamarin-dotnet-debug.dylib : xamarin_UIApplicationMain
0x103370a50 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : do_icall
0x10336f2f0 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : do_icall_wrapper
0x103363d3c - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_interp_exec_method
0x103361910 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : interp_runtime_invoke
0x10327a9b0 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_jit_runtime_invoke
0x103435a14 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_runtime_invoke_checked
0x10343cff4 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_runtime_exec_main_checked
0x1032cec70 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libmonosgen-2.0.dylib : mono_jit_exec
0x1027f7b00 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/libxamarin-dotnet-debug.dylib : xamarin_main
0x1007b6b44 - /Users/nevse/Library/Developer/CoreSimulator/Devices/4D194D78-8466-4917-80D8-F6DFF5BA52C2/data/Containers/Bundle/Application/4DF96EDA-FCD6-42F2-96EF-86DAFECBB2B3/Maui.TestApp.app/Maui.TestApp : main
0x10123d544 - Unknown
0x100be6058 - Unknown
0x4f41800000000000 - Unknown

=================================================================
Basic Fault Address Reporting
=================================================================
Memory around native instruction pointer (0x102644a4c):0x102644a3c  c0 03 5f d6 c0 03 5f d6 10 29 80 d2 01 10 00 d4  .._..._..)......
0x102644a4c  e3 00 00 54 fd 7b bf a9 fd 03 00 91 ce e2 ff 97  ...T.{..........
0x102644a5c  bf 03 00 91 fd 7b c1 a8 c0 03 5f d6 c0 03 5f d6  .....{...._..._.
0x102644a6c  70 0a 80 d2 01 10 00 d4 e3 00 00 54 fd 7b bf a9  p..........T.{..

=================================================================
Managed Stacktrace:
=================================================================
at <unknown> <0xffffffff>
at System.Reflection.RuntimeMethodInfo:InternalInvoke <0x00074>
at System.Reflection.MethodBaseInvoker:InterpretedInvoke_Method <0x00050>
at System.Reflection.MethodBaseInvoker:InvokeWithNoArgs <0x000ba>
at System.Reflection.RuntimeMethodInfo:Invoke <0x00152>
at System.Reflection.MethodBase:Invoke <0x0004e>
at <InvokeAsync>d__10:MoveNext <0x00494>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00016>
at SomeTestRunner.Engine.FixtureMethodWorkItem:InvokeAsync <0x00082>
at <InvokeMethodAsync>d__8:MoveNext <0x000e6>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00016>
at SomeTestRunner.Engine.MethodListWorkItem:InvokeMethodAsync <0x000aa>
at <RunTestAttempt>d__42:MoveNext <0x0020a>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00016>
at SomeTestRunner.Engine.WorkTreeRunner:RunTestAttempt <0x00096>
at <RunAttempts>d__41:MoveNext <0x00066>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:Start <0x00016>
at SomeTestRunner.Engine.WorkTreeRunner:RunAttempts <0x00096>
at <RunTest>d__36:MoveNext <0x00490>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start <0x00016>
at SomeTestRunner.Engine.WorkTreeRunner:RunTest <0x00096>
at <RunFixture>d__35:MoveNext <0x003ae>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start <0x00016>
at SomeTestRunner.Engine.WorkTreeRunner:RunFixture <0x00096>
at <RunNamespace>d__34:MoveNext <0x0012c>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start <0x00016>
at SomeTestRunner.Engine.WorkTreeRunner:RunNamespace <0x00096>
at <RunAssembly>d__33:MoveNext <0x0012c>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start <0x00016>
at SomeTestRunner.Engine.WorkTreeRunner:RunAssembly <0x00096>
at <RunSuite>d__32:MoveNext <0x002be>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start <0x00016>
at SomeTestRunner.Engine.WorkTreeRunner:RunSuite <0x00096>
at <Run>d__28:MoveNext <0x0015c>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start <0x00016>
at SomeTestRunner.Engine.WorkTreeRunner:Run <0x00082>
at <RunCoreAsync>d__52:MoveNext <0x00206>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start <0x00016>
at SomeTestRunner.ViewModel.MainViewModel:RunCoreAsync <0x00096>
at <RunAsync>d__50:MoveNext <0x0020e>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start <0x00016>
at SomeTestRunner.ViewModel.MainViewModel:RunAsync <0x00082>
at <OnAppearing>d__49:MoveNext <0x000da>
at System.Runtime.CompilerServices.AsyncMethodBuilderCore:Start <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder:Start <0x00016>
at SomeTestRunner.ViewModel.MainViewModel:OnAppearing <0x00082>
at <OnLoaded>d__10:MoveNext <0x0019e>
at AsyncStateMachineBox`1:ExecutionContextCallback <0x00030>
at System.Threading.ExecutionContext:RunInternal <0x00136>
at AsyncStateMachineBox`1:MoveNext <0x00110>
at AsyncStateMachineBox`1:MoveNext <0x0001e>
at <>c:<.cctor>b__17_0 <0x00026>
at System.Threading.Tasks.AwaitTaskContinuation:RunCallback <0x0008e>
at System.Threading.Tasks.SynchronizationContextAwaitTaskContinuation:Run <0x00074>
at System.Threading.Tasks.Task:RunContinuations <0x00202>
at System.Threading.Tasks.Task:FinishContinuations <0x00064>
at System.Threading.Tasks.Task`1:TrySetResult <0x0010e>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder`1:SetExistingTaskResult <0x00094>
at System.Runtime.CompilerServices.AsyncTaskMethodBuilder:SetResult <0x00072>
at <DoEvents>d__9:MoveNext <0x002c0>
at AsyncStateMachineBox`1:ExecutionContextCallback <0x00030>
at System.Threading.ExecutionContext:RunInternal <0x00136>
at AsyncStateMachineBox`1:MoveNext <0x00110>
at AsyncStateMachineBox`1:MoveNext <0x0001e>
at <>c:<.cctor>b__17_0 <0x00026>
at System.Threading.Tasks.AwaitTaskContinuation:RunCallback <0x0008e>
at System.Threading.Tasks.SynchronizationContextAwaitTaskContinuation:Run <0x00074>
at System.Threading.Tasks.Task:RunContinuations <0x00202>
at System.Threading.Tasks.Task:FinishContinuations <0x00064>
at System.Threading.Tasks.Task`1:TrySetResult <0x0010a>
at System.Threading.Tasks.TaskCompletionSource`1:TrySetResult <0x00034>
at System.Threading.Tasks.TaskCompletionSource`1:SetResult <0x00020>
at <>c__DisplayClass3_0`1:<Dispatch>b__0 <0x00052>
at <>c__DisplayClass10_0:<DispatchDelayedImplementation>b__0 <0x00028>
at CoreFoundation.DispatchQueue:static_dispatcher_to_managed <0x0010e>
at CoreFoundation.DispatchQueue:static_dispatcher_to_managed <0x0008a>
at <unknown> <0x00000>
at <unknown> <0xffffffff>
at UIKit.UIApplication:xamarin_UIApplicationMain <0x000b8>
at UIKit.UIApplication:UIApplicationMain <0x0006e>
at UIKit.UIApplication:Main <0x0015a>
at Maui.TestApp.Program:Main <0x0002c>
at <Module>:runtime_invoke_direct_void_string[] <0x0007e>
at <unknown> <0x00000>
=================================================================
```

_Copied from original issue xamarin/xamarin-macios#20190_

**Comment by thaystg** [comment]
Hi, I think this was fixed on 8.0.3 version, can you please check if I'm right?

**Comment by nevse** [comment]
yes, it works! Thank you.
